packages/nimble-blazor/NimbleBlazor/Components/NimbleMappingIcon.razor.cs

```diff
@@ -21,4 +21,10 @@ public partial class NimbleMappingIcon<TKey> : NimbleMappingBase<TKey>
     /// </summary>
     [Parameter]
     public string? Text { get; set; }
+
+    /// <summary>
+    /// Gets or sets whether or not the mapping's text should be rendered in the table's cells.
+    /// </summary>
+    [Parameter]
+    public bool TextHidden { get; set; }
```

Comment:
I think for consistency with our existing Blazor API, this should be nullable (`bool?`) - same for the property in `NimbleMappingSpinner.razor.cs`.

(I actually thought as-is this would require the property to always be set explicitly, but that doesn't look like the case, since ComponentsDemo has icon mappings without `TextHidden` specified.)